gammapy/maps/wcs.py

```diff
@@ -634,6 +635,40 @@ def solid_angle(self):
 
         return u.Quantity(dx * dy, 'sr')
 
+    def get_region_mask_array(self, region):
+        """Return mask of pixels inside region in the the form of boolean array
+
+        TODO: implement list of region for each axis
+
+        Parameters
+        ----------
+        region : `~regions.PixelRegion` or `~regions.SkyRegion` object
+            A region on the sky could be defined in pixel or sky coordinates.
+
+        Return
+        ------
+        mask_array : `~numpy.ndarray` of booleans
+            the array of
+        """
+
+        from regions import PixCoord
+        # TODO : if Pixel Compound regions are taken into account, rather convert to PixelRegion
+        # if isinstance(region, SkyRegion):
+        #    region = region.to_pixel(self.wcs)
+
+        if isinstance(region, SkyRegion):
```

Comment:
I think this would be simpler and equivalent to how regions works now:
```
if isinstance(region, SkyRegion):
    region = region.to_pixel(self.wcs)

pix = PixCoord(*self.get_idx())
return region.contains(pix)
```

Reply:
Done